src/officehours_api/consumers.py

```diff
@@ -154,7 +155,7 @@ def user(self):
 
     def _get_users(self):
         return list(
-            UserSerializer(u, context={'user': self.user}).data
+            NestedUserSerializer(u).data
             for u in User.objects.all()
```

Comment:
So, just to make sure I understand, this is faster because it's bypassing the queue-related data fetching that is in `UserSerializer` (and not in `NestedUserSerializer`)?

Reply:
Yeah, looking up the queue data in `UserSerializer` is slow can could probably be optimized. But nothing that uses this consumer needs that data anyway. I may rename `NestedUserSerializer` to `ShallowUserSerializer` or similar now that it's no longer 'just' for nesting within other serializers, but not for this patch.